### PR TITLE
Silent installation of AnyDesk (Conti)

### PR DESCRIPTION
**Pull request by j91321**
Quickly made detection for silent installation of AnyDesk Remote Software from the Conti manuals. 

Maybe can be improved with also using the Product field (so simple name name change won't break the detection).

### CODE REVIEW

File: rules/windows/process_creation/win_anydesk_silent_install.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - https://twitter.com/TheDFIRReport/status/1423361119926816776?s=20
+    - https://support.anydesk.com/Automatic_Deployment
+tags:
+    - attack.t1219
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_anydesk:
+        CommandLine|contains: 'anydesk.exe'
```
Comment (frack113) [inline rules/windows/process_creation/win_anydesk_silent_install.yml:17]: As `OriginalFileName` is "-" 
```Yaml
Product: AnyDesk
```

File: rules/windows/process_creation/win_anydesk_silent_install.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    selection_install:
+        CommandLine|contains: '--install'
+    selection_with_win:
+        CommandLine|contains: '--start-with-win'
+    selection_silent:
+        CommandLine|contains: '--silent'
+    condition: all of them
```
Comment (frack113) [inline rules/windows/process_creation/win_anydesk_silent_install.yml:24]: Why not
```yaml
selection:
    Product: AnyDesk
    CommandLine|contains|all:
        '--install '
        '--start-with-win '
        '--silent '
condition: selection
```

**Review by frack113 — CHANGES_REQUESTED** [review]

File: rules/windows/process_creation/win_anydesk_silent_install.yml (hunk 2 above)
Comment (Neo23x0) [reply rules/windows/process_creation/win_anydesk_silent_install.yml:24]: I'd create two sections since not all log sources contain the OriginalFileName

Comment (Neo23x0) [reply rules/windows/process_creation/win_anydesk_silent_install.yml:21]: --start-with-win is pretty specific - a quick Google search didn't return many FPs. Combined with the other two it should be good enough.

Comment (j91321) [reply rules/windows/process_creation/win_anydesk_silent_install.yml]: I had the OriginalFilename field empty for AnyDesk executable.  I did want to use the Product field, but noticed it's not mapped in some outputs. Two sections are probably the best way to go.

**Review by frack113 — APPROVED** [review]

**Pull request merged**